ansible/setup.yml

```diff
@@ -0,0 +1,19 @@
+- name: Common configuration for all servers
+  hosts: all
+  become: true
+  remote_user: root
+  roles:
+    - system
+    - docker
+    - postgres
+    - redis
```

Comment:
The `wtsh-hypha` image currently hardcodes the `CACHES` settings to use the database backend, so technically `redis` is not used (yet).